.ci/run_nccl_test.sh

```diff
@@ -0,0 +1,356 @@
+#!/bin/bash -leE
+
+SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+echo "DEBUG: SCRIPT_DIR = ${SCRIPT_DIR}"
```

Comment:
same

Reply:
SCRIPT_DIR is needed to correctly source settings.sh by absolute paths - verified with main script called from folders on different levels and even when script is called with relative paths.